City-of-Pori/pori-kada
# PORI-406 Added configuration changes to enable section selection to l…

…anding pages

To test:

1. drush fra -y && drush cc all
2. Go to any visit pori landing page content such as https://local.visitpori.fi/kirjurinluoto and edit it
3. Make sure you are able to choose section for it under "Omat sivustot" tab. Choose a section.
4. Save the node and make sure that the footer written in the section node appears on the node footer.
